kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/CSVUtil.java

```diff
@@ -16,29 +16,35 @@
 
 package org.kitesdk.data.spi.filesystem;
 
-import au.com.bytecode.opencsv.CSVParser;
-import au.com.bytecode.opencsv.CSVReader;
-import au.com.bytecode.opencsv.CSVWriter;
-import com.google.common.collect.ImmutableSet;
-import java.util.Set;
-import javax.annotation.Nullable;
-import org.apache.avro.SchemaBuilder;
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.CharMatcher;
-import com.google.common.base.Preconditions;
-import com.google.common.collect.Lists;
+import static java.lang.Math.min;
+
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.io.OutputStreamWriter;
 import java.nio.charset.Charset;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Set;
 import java.util.regex.Pattern;
+
```

Comment:
Nit: it looks like your editor is reordering imports and adding blank lines. That makes it difficult to backport or work with these changes so I'd appreciate it if you could keep this to just functional changes.